src/app/organization/organization.component.html

```diff
@@ -100,6 +100,16 @@ <h4 matLine>Password Preferences</h4>
               <p matLine>Define standards for enforcing the usage of stronger passwords</p>
             </mat-list-item>
           </div>
+
```

Comment:
@Onyx2406 Is there a reason why the bulkimport was moved?

Reply:
@adamsaghy I've moved "bulkimport" from the right to the left to ensure frontend consistency. Let me know if you'd like me to move it back to its original position, but if I do, the left block will remain blank.
Check this: https://i.imgur.com/HOETIdL.png